learn/safe-core/safe-core-account-abstraction-sdk/onramp-kit/integrate-monerium-and-safe.md

```diff
@@ -0,0 +1,184 @@
+# How to integrate Monerium with your Safe
+
+The [`MoneriumPack`](https://github.com/safe-global/safe-core-sdk/tree/main/packages/onramp-kit/src/packs/monerium) enables Safe users to make direct transfers of digital money from their Safe addresses to an IBAN via the SEPA network. This allows them to use Monerium and Safe services together.
+
+More info about Monerium:
+
+- [Monerium](https://monerium.com)
+- [Monerium Developers](https://monerium.dev)
+
+### What are we going to learn?
+
+This guide demonstrates how to use the `MoneriumPack` with the [`OnRampKit`](https://github.com/safe-global/safe-core-sdk/tree/main/packages/onramp-kit) and incorporate it into your web application.
+
+### Prerequisites
+
+1. [Node.js and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+2. [Monerium account and application](https://monerium.dev/docs/getting-started/create-app)
+3. A web application using your favorite CLI and language. We recommend:
+
+- [Vite for React projects](https://vitejs.dev/guide/#scaffolding-your-first-vite-project)
+- [Vue CLI](https://cli.vuejs.org/guide/creating-a-project.html#vue-create)
+- [Angular CLI](https://angular.io/cli/new)
+
+Monerium offers several authentication methods for web apps. The `MoneriumPack` implements the _Authorization code flow with proof key for code exchange (PKCE)_. Learn more about it [here](https://monerium.dev/docs/api#authentication).
+
+### Install dependencies
+
+```bash
+yarn add @safe-global/onramp-kit @monerium/sdk
```

Comment:
```suggestion
yarn add @safe-global/onramp-kit @monerium/sdk @safe-global/protocol-kit
```

Reply:
mmm yes, for instantiating safeSdk they are always going to require the protocol-kit